src/Symfony/Bundle/WebProfilerBundle/Resources/config/schema/webprofiler-1.0.xsd

```diff
@@ -8,7 +8,14 @@
     <xsd:element name="config" type="config" />
 
     <xsd:complexType name="config">
-        <xsd:attribute name="toolbar" type="xsd:boolean" />
+        <xsd:sequence>
+            <xsd:element name="toolbar" type="toolbar" />
```

Comment:
I guess it is optional? Then it could be like this:
```suggestion
            <xsd:element name="toolbar" type="toolbar" minOccurs="0" maxOccurs="1" />
```
But for this single item no need for a `sequence` ... so that could be just removed. minOccurs and maxOccurs are by default `1`. So when it should always be there just the way you have it without the unnecessary `sequence` wrapper.
Sequence is to define a specific order of a list of elements.

Reply:
@maxbeckers AFAIK, we cannot define child elements without using `sequence` or `choice` around them. And it is better to use `sequence` then (`choice` would not work when we add a second child node).

However, the bounds for occurrences are indeed needed.

Reply:
@stof AFAIK it is not required to have the `sequence` (or `choice`) around ... but not 100% sure so maybe just keep the `sequence`, doesn't break anything ... also from the [spec](https://www.w3.org/TR/xmlschema-1/#element-complexType) not 100% clear to me

Reply:
kept the sequence